src/Sylius/Behat/Context/Ui/Admin/ManagingShippingMethodsContext.php

```diff
@@ -12,87 +12,428 @@
+     */
+    public function thereShouldStillBeOnlyOneShippingMethodWith($element, $code)
+    {
+        $this->iWantToBrowseShippingMethods();
+
+        Assert::true(
+            $this->indexPage->isResourceOnPage([$element => $code]),
+            sprintf('Shipping method with %s %s cannot be founded.', $element, $code)
+        );
+    }
+
+    /**
+     * @Given I want to modify a shipping method :shippingMethod
+     * @Given /^I want to modify (this shipping method)$/
+     */
+    public function iWantToModifyAShippingMethod(ShippingMethodInterface $shippingMethod)
+    {
+        $this->updatePage->open(['id' => $shippingMethod->getId()]);
+    }
+
+    /**
+     * @Then the code field should be disabled
+     */
+    public function theCodeFieldShouldBeDisabled()
+    {
+        Assert::true(
+            $this->updatePage->isCodeDisabled(),
+            'Code should be immutable, but it does not.'
+        );
+    }
+
+    /**
+     * @Then /^(this shipping method) name should be "([^"]+)"$/
+     * @Then /^(this shipping method) should still be named "([^"]+)"$/
+     */
+    public function thisShippingMethodNameShouldBe(ShippingMethodInterface $shippingMethod, $shippingMethodName)
+    {
+        $this->iWantToBrowseShippingMethods();
+
+        Assert::true(
+            $this->indexPage->isResourceOnPage(
+                [
+                    'code' => $shippingMethod->getCode(),
+                    'name' => $shippingMethodName,
+                ]
+            ),
+            sprintf('Shipping method name %s has not been assigned properly.', $shippingMethodName)
+        );
+    }
+
+    /**
+     * @When I save my changes
+     * @When I try to save my changes
+     */
+    public function iSaveMyChanges()
+    {
+        $this->updatePage->saveChanges();
+    }
+
+    /**
+     * @Then I should be notified about successful edition
+     */
+    public function iShouldBeNotifiedAboutSuccessfulEdition()
+    {
+        $this->notificationChecker->checkEditionNotification(self::RESOURCE_NAME);
+    }
+
+    /**
+     * @Then I should be notified that :element is required
+     */
+    public function iShouldBeNotifiedThatIsRequired($element)
+    {
+        $this->assertFieldValidationMessage($element, sprintf('Please enter shipping method %s.', $element));
+    }
+
+    /**
+     * @Then shipping method with :element :name should not be added
+     */
+    public function shippingMethodWithElementValueShouldNotBeAdded($element, $name)
+    {
+        $this->iWantToBrowseShippingMethods();
+
+        Assert::false(
+            $this->indexPage->isResourceOnPage([$element => $name]),
+            sprintf('Shipping method with %s %s was created, but it should not.', $element, $name)
+        );
+    }
+
+    /**
+     * @When I do not name it
+     */
+    public function iDoNotNameIt()
+    {
+        // Intentionally left blank to fulfill context expectation
```

Comment:
No. Because otherwise I would have to specify language of it. So I decided, to leave it blank